pipeline/ruleset.go

```diff
@@ -57,6 +58,7 @@ type (
 		Status   string   `json:"status,omitempty"  yaml:"status,omitempty"`
 		Tag      string   `json:"tag,omitempty"     yaml:"tag,omitempty"`
 		Target   string   `json:"target,omitempty"  yaml:"target,omitempty"`
+		Label    string   `json:"label,omitempty"   yaml:"label,omitempty"`
```

Comment:
Thoughts on extending this rule to include other `pull_request` events, which would result in passing in a slice of label strings? So if a PR has labels `bug` and `v2`, we'd pass in a label set of `[]string{"bug", "v2"}`?